update

```diff
@@ -21,6 +21,13 @@ fs.readdir(path.join(__dirname, '/bin'), function (err, files) {
     if (build.prerelease && build.prerelease.length > 0) {
       version += '-' + build.prerelease
     }
+
+    // workaround for pre-0.4.0, where prereleases didn't bump the version
+    // NOTE: the text must be lexically after 'nightly'
+    if (semver.major(build.version) === 0 && semver.minor(build.version) < 4) {
+      version += '-workaround'
```

Comment:
I think it would be better to change the order in the file without actually modifying the content of the fields in JSON.

Reply:
Which file?

Reply:
Ah, sorry, I misread it. I thought it changes the `version` field value that goes into the JSON but now I see that this is just a local variable that is returned from the function.

Reply:
Yes, it is a trick for the internal `sort()` to work in a way we want.